.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 79
+select = B,C,E,F,W,T4,B9,B950
+ignore = E402, W503, W504
```

Comment:
This overrides existing `flake8` config present in `tox.ini`.

Reply:
Yes, that's why the fast tests were failing on the original branch, until I added our usual warning ignore settings here instead of the one @sgaist used.

As per my comments above, `pycodestyle` (and `flake8`, which wraps `pycodestyle`) ignores the `tox.ini` config on one of my dev platforms, because of inline comments in the file, even though the `pycodestyle` version claims to be the same. 

However, now that I understand the cause I could move those comments, and we could drop this new config.  Is there any reason at all to have both?

Annoyingly, it seems `tox.ini` needs duplicate pycodestyle settings, under `[pycodestyle]` and `[flake8]` sections, unless you never run `pycodestyle` alone.  Also, flake8 ignores W503 and W504 by default (but not if you explicitly ignore anything else).

Reply:
There's a bit of complexity with all the possible configuration files that can be used. There's also the pyproject that might be used but may not be supported by all the tools.

I have a project which has flake8 in `tox.ini`, bandit in its own file because they were (might still) not supporting the `pyproject.toml` nor `tox.ini`, and the rest is in `pyproject.toml`. I like the idea of having a maximum of configurations in a single file but that is rather up to the taste of the maintainer(s).